content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -1,74 +1,191 @@
 # Run rippled as a Validator
 
-Running a `rippled` validator that participates in the Consensus process is simple:
+A `rippled` server running in validator mode does everything a stock server does: ***TODO: I see stock servers referred to as "tracking servers" too - should we always call them one or the other? "Tracking servers" is a little more descriptive and it used in the Consensus article: https://developers.ripple.com/consensus.html. Or would it be confusing because "tracking" is also a server state: 	The server is in agreement with the network?***
 
-1. [Enable validation](run-rippled-as-a-validator.html) on your `rippled` server.
-    * At first, your server is an _untrusted validator_. Others can see the validations your server issues, but they disregard them in the consensus process.
-2. Share the public key with the public, especially other `rippled` operators.
-3. When other `rippled` operators add your public key to their list of trusted servers, you have become a _trusted validator_.
-    * Also see [Properties of a Good Validator](rippled-server-modes.html#properties-of-a-good-validator) for best practices.
+- Connects to a network of peers
 
+- Relays cryptographically signed transactions
 
-## Validator Setup
+- Maintains a local copy of the complete shared global ledger
 
-The `validator-keys` tool (included in the `rippled` RPM) is the recommended means to securely generate and manage your validator keys.
+What makes a validator _different_ is that it also issues validation messages, which are sets of candidate transactions for evaluation by the XRP Ledger network during the [consensus process](consensus-principles-and-rules.html#how-consensus-works).
 
-1. [Install a `rippled` server.](install-rippled.html)
+It's important to understand that these validation messages are used only by `rippled` servers that explicitly trust your validator by listing it on their Unique Node Lists (UNLs). Until your validator is listed on a UNL, it is considered an _untrusted_ validator and its validation messages are disregarded in the consensus process.
 
-2. Generate a validator key pair:
+For this reason, aside from getting your validator up and running, another key aspect of operating a validator is putting infrastructure in place that engenders trust in your validator. Before validation list publishers can add your validator to their UNLs, they must first have trust in your validator. Once your validator has been added to a UNL, it is a _trusted_ validator that participates in the consensus process. ***TODO: what do we mean by "validator list publishers"? Is this specifically about getting a validator listed in this blob? https://vl.ripple.com/ Why is it in a blob and can we provide guidance about how to decode it? Whatever is in this list would be reflected on XRPCharts' list of UNL validators, correct?***
```

Comment:
Anyone can publish a list to follow. Ours is the default list and the one that is trusted by most of the network. Nodes should follow lists that are very similar to avoid forking.

Reply:
Fantastic call on replacing "engenders" with "builds" 🙌 ... and thank you for taking the time to answer my questions!